corehq/apps/app_manager/static/app_manager/json/case-reserved-words.json

```diff
@@ -17,7 +17,9 @@
     "date_opened",
     "doc_type",
     "domain",
-    "external-id",
```

Comment:
Just out of curiosity, I see that we [list both](https://github.com/dimagi/commcare-hq/blob/b6c48c2a7ef3fcc1e427692bcd8d78f902b50128/corehq/apps/app_manager/static/app_manager/json/case-reserved-words.json#L16) `date-opened` and `date_opened`; why is that (is it related to [this logic](https://github.com/dimagi/commcare-hq/blob/802543c60468d8fba80f2c0c4f44e34a66915d1c/corehq/apps/app_manager/detail_screen.py#L23))? And should we be doing that for external_id?

Reply:
Nicely spotted, and excellent question!

It looks like `CASE_PROPERTY_MAP` is only used once, to determine the XPath for a case property, used by the case details screen in a CommCare app. `CASE_PROPERTY_MAP`'s comment refers to `corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js` but that file doesn't seem to exist any more.

... So I don't _think_ `CASE_PROPERTY_MAP` is related to this. I don't know why `date-opened` is listed here. Maybe @snopoke or @dannyroberts know?